site/layouts/partials/announcement.html

```diff
@@ -0,0 +1,17 @@
+<div class="d-block px-3 py-2 text-center text-bold bd-announcement">
+  Bootstrap 4 has reached end of life.
+  <a
+    href="https://getbootstrap.com/docs/5.3/migration/"
+    target="_blank"
+    rel="noopener"
+    >Upgrade to the latest</a
+  >
+  or consider
+  <a
+    href="https://www.herodevs.com/support/nes-bootstrap?utm_source=Bootstrap_site&utm_medium=Banner&utm_campaign=v4_eol"
```

Comment:
I'm wondering if it wouldn't be better for the users to have a link to the new `end-of-life` page to have some information, and then they can decide whether or not they want to go to the HeroDevs website.

Reply:
@edezekiel Any thoughts on this from the HeroDevs side? I too am wondering about linking to this new page. We could add a third link to the banner, but I imagine that's some diminishing returns and fatigue to users.